.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!docker/
+!pkg/docker
```

Comment:
I ask because AFAICS the `.dockerignore` file correctly ignores both `docker/` and `pkg/docker`.